config.go

```diff
@@ -130,6 +133,7 @@ type ExitConfiguration struct {
 	TcpDialContext                 func(ctx context.Context, network, addr string) (net.Conn, error) `json:"-"`
 	HttpDialContext                func(ctx context.Context, network, addr string) (net.Conn, error) `json:"-"`
 	WsDialContext                  func(ctx context.Context, network, addr string) (net.Conn, error) `json:"-"`
+	MinBalance                     float64                                                           `json:"minBalance"`
```

Comment:
Should be `ReverseMinBalance` because it's used in reverse mode

Reply:
Yes, modified